src/Components/Task-editor.tsx

```diff
@@ -0,0 +1,63 @@
+import { inject, observer } from "mobx-react";
+import React from "react";
+import TaskItem from "../Models/TaskItem";
+import TaskStore from "../Stores/TaskStore";
+
+@inject('TaskStore')
+@observer
+export default class TaskEditor extends React.Component<{ taskStore: typeof TaskStore }> {
+    textTask: any;
+
+    addTodo = (arg0: { title: string; completed: boolean; }) => {
+        debugger;
```

Comment:
debuggers should not commit to pr,
what you can do is introduce `linter` with `githooks` to prevent people do these mistakes.
Try out for an exercise.

Reply:
isInvalid